tests/test_coordinated_workers/test_coordinator.py

```diff
@@ -239,3 +241,39 @@ def test_s3_integration(
         assert coordinator.s3_connection_info.tls_ca_chain == tls_ca_chain
         assert coordinator._s3_config["endpoint"] == endpoint_stripped
         assert coordinator._s3_config["insecure"] is (not tls_ca_chain)
+
+
+def test_tracing_receivers_urls(coordinator_state: State, coordinator_charm: ops.CharmBase):
+
+    charm_tracing_relation = Relation(
+        endpoint="my-charm-tracing",
+        remote_app_data={
+            "receivers": json.dumps(
+                [{"protocol": {"name": "otlp_http", "type": "http"}, "url": "1.2.3.4:4318"}]
+            )
+        },
+    )
+    workload_tracing_relation = Relation(
+        endpoint="my-workload-tracing",
+        remote_app_data={
+            "receivers": json.dumps(
+                [
+                    {"protocol": {"name": "otlp_http", "type": "http"}, "url": "1.2.3.4:4318"},
```

Comment:
The assumption is that it shouldn't matter as its the same endpoint. i.e if workload and charm tracing both request otlp_http, it should be the same endpoint returned. right?

Reply:
well no, what if you relate charm and workload tracing to two different tempo instances?

Reply:
Right. In that case, I think we should split the databag into 2 fields `charm_tracing_receivers` and `workload_tracing_receivers` to avoid the collision situation.